docs/adding_standalone_benchmark.md

```diff
@@ -0,0 +1,58 @@
+# Adding Standalone Benchmarks in vSwarm
```

Comment:
please move to a separate PR, ask David for a review. Also, please run by Manyi first.